sdk/lib/drivers/wdf/shared/core/fxdevice.cpp

```diff
@@ -1332,6 +1332,7 @@ FxDevice::_AcquireOptinRemoveLock(
 
 _Must_inspect_result_
 NTSTATUS
+STDCALL
```

Comment:
Why nobody complains about that right now? Are these functions not used?

Reply:
My guess is that adding more attributes leads to better checks.